### ISSUE TITLE
clarification to CONTRIBUTING doc

### ISSUE THREAD

**Issue by uvtc** [janet-lang/pkgs#47]
Two things I think need to be clarified:

 1. Regarding the symbol that goes into pkgs.janet, should it be the same as the `:name` value in the project's `declare-project` (in the project's project.janet file)?

 2. The contributing doc says, "One exception is the janet- prefix in the git repository name to avoid name clashes.", but does not explain the meaning. I *think* what it means to say is, if your git repo name includes the "janet-" prefix, don't include that in the project name. Is that correct?

**Comment by bakpakin** [comment]
Yes, the naming is really more about keeping consistency. As far as what jpm uses the names for, yes they are aliases to git repos. They are more important than the :name field in a declare project as far as jpm is concerned.
